### DISCUSSION TITLE
per app support

### DISCUSSION THREAD

**Discussion by mobycloud** [gravitational/teleport#9653] [General]
Can Teleport give a per app support? For example when use it for reverse SSH uses a split tunnel on Teleport which does not affect other type of traffic that is critical to delays, for example, a media streaming traffic.

For example, most VPNs by default take over the OS and all runnings apps must go through the VPN. But reverse SSH is a distinct process for itself- it is an individual tunnel that does not affect other apps.

**Comment by zen** [comment]
It's quite hard to understand what you are asking for. If you want some sort of traffic shaping, you can use operating system tools.
